src/devtools/client/debugger/src/components/Editor/LineHitCountsToggle.tsx

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import { useFeature } from "ui/hooks/settings";
+
+import Toggle from "./Toggle";
+
+function LineHitCountsToggle() {
+  const { value } = useFeature("hitCounts");
+
+  if (!value) {
+    return null;
+  }
+
+  return (
+    <label className="flex items-center space-x-1 pt-0.5 pl-3">
+      <Toggle enabled={true} setEnabled={() => {}} />
```

Comment:
TODO: I need to add this state to the Redux store.

Reply:
We re-visiting the toggle?

Reply:
I had it at first but we decided to remove it.

Looks like this maybe shouldn't have been committed? since it's not actually being used?

Reply:
Ooops looks like this was already removed and I was commenting on a stale thing. My bad!